website/docs/how-to/how-to-import-export.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: '[Deprecated] Import & Export'
+title: 'Import & Export'
```

Comment:
Shouldn't this be still deprecated?

> Api admin state is deprecated from version 5 and removed from version 6. We recommend using the new [Environment Import & Export](https://docs.getunleash.io/reference/deploy/environment-import-export).

Reply:
Ok, I think this is just style changing on how we communicate that this is actually deprecated :+1: